rtsf-at-checkout-loss-detector/Dockerfile

```
@@ -17,15 +17,15 @@ RUN go mod download
 
 COPY . .
 
-RUN go build -tags no_zmq -o loss-detector /loss-detector/main.go
+RUN CGO_ENABLED=0 go build -o loss-detector /loss-detector/main.go
 
 # Next image - Copy built Go binary into new workspace
-FROM alpine:3.16
+FROM alpine:3.17
 
 LABEL license='SPDX-License-Identifier: BSD-3-Clause' \
-    copyright='Copyright (c) 2019 - 2022: Intel'
+    copyright='Copyright (c) 2019 - 2023: Intel'
 
 COPY --from=builder /loss-detector/res/ /res/
 COPY --from=builder /loss-detector/loss-detector /loss-detector
 
-CMD [ "/loss-detector", "--cp=consul://edgex-core-consul:8500", "--registry", "--confdir=/res"]
+CMD [ "/loss-detector", "--cp=consul://edgex-core-consul:8500", "--registry", "--configDir=/res"]
```

`/res` is the default, so remove the redundant flag.

```suggestion
CMD [ "/loss-detector", "--cp=consul://edgex-core-consul:8500", "--registry"]
```